packages/grid/src/index.scss

```diff
@@ -15,11 +15,9 @@ $grid-gutter-width: 20px !default;
 @import 'bootstrap/scss/functions';
 @import 'bootstrap/scss/variables';
 @import 'bootstrap/scss/mixins/breakpoints';
-@import 'bootstrap/scss/mixins/grid-framework';
```

Comment:
Looks like this has been [consolidated](https://github.com/twbs/bootstrap/pull/29146/files#diff-41667d8b9901aa9fa52483b538bb9026c287f2c663d2fdc01acffa06888cc087R52) into `grid`.